src/Microsoft.Identity.Client/Platforms/iOS/EmbeddedWebview/WKWebNavigationDelegate.cs

```diff
@@ -50,7 +50,7 @@ public override void DecidePolicy(WKWebView webView, WKNavigationAction navigati
                    requestUrlString.StartsWith(iOSBrokerConstants.BrowserExtInstallPrefix, StringComparison.OrdinalIgnoreCase))
             {
                 AuthenticationAgentUIViewController.DismissViewController(true, () =>
-                    AuthenticationAgentUIViewController.callbackMethod(new AuthorizationResult(AuthorizationStatus.Success, requestUrlString)));
+                    AuthenticationAgentUIViewController.callbackMethod(AuthorizationResult.FromUri(requestUrlString)));
```

Comment:
nit:  callbackMethod should be named CallbackMethod.  publicly accessible types should be PascalCased.